CHANGELOG.md

```diff
@@ -97,6 +97,8 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * ``Sequence`` objects now handle slicing of empty positional metadata correctly. Any metadata that is empty will no longer be propagated by the internal ``_to`` constructor. ([#1133](https://github.com/biocore/scikit-bio/issues/1133))
 * ``DissimilarityMatrix.plot()`` no longer leaves a white border around the
   heatmap it plots (PR #1070).
+* TreeNode.root_at_midpoint`` no longer fails when a node with two equal length child branches exists in the tree. ([#1077](https://github.com/biocore/scikit-bio/issues/1077))
+* ``TreeNode._set_max_distance`` would store distance information as ``list``s in the attribute ``MaxDistTips`` on each node, however, these distances were only valid for the node in which the call to ``_set_max_distance`` was called on. ([#1223](https://github.com/biocore/scikit-bio/issues/1223))
```

Comment:
My previous comment on this note may have been lost, but can you update to note which public methods were effected by this so users know what results they'd previously generated may have been incorrect? Users won't know how this bug impacts them since we don't expect them to know what private functionality they've accessed (e.g., I don't know how it's impacted me). 

For the same reasons we are talking about below, public methods which were impacted by this should also have new tests added (the fact that this wasn't caught means that there was a condition that those methods weren't testing).